pages/standards-guidelines/wcag/non-web-ict.md

```diff
@@ -44,24 +44,64 @@ Quick link:
 
 ## Introduction {#intro}
 
-<cite>Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)</cite> describes how the Web Content Accessibility Guidelines (WCAG) can be applied to non-web information and communications technologies (ICT), including documents and software. WCAG2ICT is a W3C Group Note that provides informative guidance (not normative requirements).
+<cite>Guidance on Applying WCAG 2 to Non-Web Information and Communications Technologies (WCAG2ICT)</cite> describes how the Web Content Accessibility Guidelines (WCAG) can be applied to non-web information and communications technologies (ICT), including documents and software. WCAG2ICT is the acronym used for the W3C Guidance on Applying WCAG 2 to Non-web Information and Communication Technology Group Note.
+
+The WCAG2ICT Group Note provides:
+* [Informative](https://www.w3.org/TR/WCAG22/#dfn-informative) **guidance** that is not [normative](https://www.w3.org/TR/WCAG22/#dfn-normative) and that does not set requirements.
+* **Comments on applicability** of the Web Content Accessibility Guidelines (WCAG) when applied to non-web information and communications technologies (ICT) - this includes  non-web documents and software, including mobile apps, native applications, platform software, and [software on ICT with closed functionality[(https://www.w3.org/TR/wcag2ict/#comments-on-closed-functionality).
```

Comment:
"applicability of the Web Content Accessibility Guidelines (WCAG) when applied to" sounds weird to me. Why not just "guidance on how WCAG could apply to"?

```suggestion
* **Guidance** on how the Web Content Accessibility Guidelines (WCAG) could apply to non-Web information and communications technologies (ICT) - this includes  non-Web documents and software, including mobile apps, native applications, platform software, and software on ICT with closed functionality.
```

Reply:
I know it sounds a little weird, but I think with our new scope of work, we'll have more applicability notes than simple guidance on applying.  Would prefer it that part be left as-is.

Regarding the capitalization of "web". I thought this was no longer being done except when referring to the Web itself. WCAG 2.2 updates last December removed that capitalization anytime "web" was used as an adjective, such as "web content". Does the WAI website have different capitalization rules?

I'm also OK with removing the link to the closed functionality section.